# Default initial positions for arctos's ros2_control fake system

initial_positions:
  A_joint: 0.0
  B_joint: 0.0
  C_joint: 0.0
  Left_jaw_joint: 0.0
  X_joint: 0.0
  Y_joint: 0.0
  Z_joint: 1.57